src/components/FriendsList/FriendsList.scss

```diff
@@ -0,0 +1,204 @@
+.friends-list {
+  height: 100vh;
+  width: 400px;
+  padding: 5px;
+  box-sizing: border-box;
+  background-color: var(--color-grey-dark-01);
+  display: flex;
+  flex-direction: column;
```

Comment:
CSS Order https://9elements.com/css-rule-order/

(nu stiu mie mi se pare prea bine la inceput sa stii position si display-ul, ca sa stii cam cum e asezat in pagina)
```
  display: flex;
  flex-direction: column;
  width: 400px;
  height: 100vh;
  padding: 5px;
  box-sizing: border-box;
  background-color: var(--color-grey-dark-01);
```